main.go

```diff
@@ -23,14 +23,32 @@ func main() {
 			Usage: "Fetch a Bugzilla issue by its ID.",
 			Flags: []cli.Flag{
 				&cli.StringSliceFlag{
-					Name:     "id",
-					Required: true,
+					Name: "id",
+				},
+				&cli.StringSliceFlag{
+					Name: "keywords",
+				},
+				&cli.StringSliceFlag{
+					Name: "limit",
 				},
 			},
 			Action: func(c *cli.Context) error {
-				ids := strings.Split(c.StringSlice("id")[0], ",")
-				for _, id := range ids{
-					resp := commands.GetBug(id)
+				var (
+					filters  []string
+					limit string = "5"
+				)
+
+				if c.IsSet("id") {
+					filters = strings.Split(c.StringSlice("id")[0], ",")
```

Comment:
I wonder if we could use a regex looking for at least one ID (i.e., digits) that could be separated by commas? The results should give us a list of IDs, I think.

Reply:
I don't understand, can you elaborate, give an example maybe?

Reply:
```golang
package main

import (
    "regexp"
    "fmt"
)

func main() {
    var re = regexp.MustCompile(`(?m)[0-9]{1,}`)
    var str = `'876656878'
'12355,67898'`
    
matched := re.FindAllString(str, -1)
    fmt.Println(matched)
}
```

Output

```
[876656878 12355 67898]
```